docs/trouble_shooting.md

```diff
@@ -0,0 +1,20 @@
+# Trouble shooting
+
+## get log of clusterapi-controllers containers
+
+This guide (based on minikube and others should be similar) explains general info on how to debug issues if cluster failed to create.
```

Comment:
How about move this between `Trouble shooting` and `Get log of clusterapi-controllers containers`?